library/src/main/java/com/google/android/exoplayer/text/eia608/Eia608Parser.java

```diff
@@ -153,6 +156,12 @@
         continue;
       }
 
+      // Ignore repeated control characters.
+      if (ccData1 == previousCtrl1 && ccData2 == previousCtrl2) {
```

Comment:
I completely agree about the first point, zsmatyas. I somehow expected all values before 0x10 to be covered, but they don't seem to be. I will update the change set to fix this.
I'm not sure about the second point. While I have a reference recommending the repeated messages, I have not been able to find any information on how to deal with these repeated messages. Can we expect a maximum of two or are more allowed?